spec/locales-currencies-tz.html

```diff
@@ -60,11 +60,7 @@ <h1>CanonicalizeLanguageTag ( _locale_ )</h1>
 
```

Comment:
Suggested changes in IsStructurallyValidLanguageTag 
```
"The abstract operation returns true if _locale_ can be generated from the ABNF grammar
 -  in section 2.1 of the RFC, starting with Language-Tag, 
 +  in section 3.2 of the UTS35, starting with unicode_locale_id
and does not contain duplicate variant or singleton subtags (other than as a private use subtag). It returns false otherwise. Terminal value characters in the grammar are interpreted as the Unicode equivalents of the ASCII octet values given."
```

Reply:
Done